Create ui-c8y-1020-1-0-show-assets-and-groups-on-list-for-edit-type-dashboard-instance.md (#2200)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1020-1-0-show-assets-and-groups-on-list-for-edit-type-dashboard-instance.md

---
date: ""

> title: Show assets and groups on list for edit type dashboard instance (#6351) [GRAFT][release/cd] (#6573)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Assets and groups are now shown when editing dashboard instances
```

product_area: Application enablement & solutions
change_type:

>   - value: change-QHu1GdukP

Reviewer: ```suggestion
  - value: change-VSkj2iV9m 
```

>     label: Feature

Reviewer: The description actually sounds more like a fix than a feature @jdreSAG .

>     label: Feature

Reviewer 2: ```suggestion
    label: Fix
```

component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-59255
version: 1020.1.0
---

> Show assets and groups on list for edit type dashboard instance (#6351) [GRAFT][release/cd] (#6573)

Reviewer 2: ```suggestion
Dashboard manager is showing dashboards for both devices and groups/assets type dashboards (for the latter one they could be created when application options hideTypeDashboardForAssets=false), but when you click 'Edit', new modal was showing target assets only for devices dashboards, but not for groups/assets (for the groups/assets list was empty). Now, when you click Edit, list was always populated with devices and groups/assests.
```

> Show assets and groups on list for edit type dashboard instance (#6351) [GRAFT][release/cd] (#6573)

Author: The Claude AI suggested to use the following body:
```suggestion
Previously, when editing a dashboard instance, the lists of assets and groups were not displayed, making it difficult for users to see which assets and groups were associated with the dashboard instance. With this change, the lists of assets and groups are now shown when editing a dashboard instance. This improves the user experience by providing more visibility into the configuration of the dashboard instance, allowing users to easily review and modify the associated assets and groups as needed.
```
